.github/workflows/python-integration-tests.yml

```diff
@@ -121,21 +121,6 @@ jobs:
       - name: Install dependencies
         run: |
           uv sync --all-extras --dev
-      - name: Install Ollama
```

Comment:
You may have to move the ollama env variables to the job level. Now they are in the workflow level.

Reply:
that's not an issue

Reply:
Nvm, I see you excluded ollama in the test run